src/databricks/labs/ucx/azure/resources.py

```diff
@@ -245,15 +348,17 @@ def subscriptions(self):
                 continue
             yield subscription
 
-    def storage_accounts(self) -> Iterable[AzureResource]:
+    def storage_accounts(self) -> Iterable[StorageAccount]:
```

Comment:
@nfx : Updated this return type to yield StorageAccount. Needed the `.location` when creating the access connector.